Mathlib/Init/Data/List/Lemmas.lean

```diff
@@ -72,9 +172,8 @@ theorem length_mapAccumr₂ :
   | f, _ :: x, _ :: y, c =>
     calc
       succ (length (mapAccumr₂ f x y c).2) = succ (min (length x) (length y)) :=
-        congrArg succ (length_mapAccumr₂ f x y c)
+        congr_arg succ (length_mapAccumr₂ f x y c)
```

Comment:
Out of scope, but: why do we have both `congr_arg` and `congrArg`?

Reply:
I believe `congrArg` is in lean core, but mathlib added `congr_arg` as an alias that matches the naming convention.